test/selenium/pages/OneOffContributionthankYou.scala

```diff
@@ -0,0 +1,17 @@
+package selenium.pages
+
+import org.scalatest.selenium.Page
+import selenium.util.{Browser, Config}
+
+object OneOffContributionThankYou extends Page with Browser {
+
+  val url = s"${Config.supportFrontendUrl}/contribute/one-off/thankyou"
```

Comment:
Is there any way of reverse-routing this from the routes file, to force a compilation error if it changes?

Reply:
Personally, I prefer that if we change the URL the test should fail so the developer is aware that is a breaking change.